### PR TITLE
fix markdown syntax issues in the 2.7 migration guide

### PR DESCRIPTION
**Pull request by limpid-kzonix**
# Pull Request Checklist

* [x] Have you read [How to write the perfect pull request](https://github.com/blog/1943-how-to-write-the-perfect-pull-request)?
* [x] Have you read through the [contributor guidelines](https://www.playframework.com/contributing)?
* [x] Have you signed the [Lightbend CLA](https://www.lightbend.com/contribute/cla)?
* [x] Have you referenced any issues you're fixing using [commit message keywords](https://help.github.com/articles/closing-issues-using-keywords/)?
* [x] Have you added copyright headers to new files?
* [x] Have you checked that both Scala and Java APIs are updated?
* [x] Have you updated the documentation for both Scala and Java sections?
* [x] Have you added tests for any changed functionality?

# Helpful things

## Purpose

Fix markdown formatting issues in the 2.7 migration guide 

## Background Context

Why did you take this approach?

The documentation page is not displayed correctly due to markdown formatting issues

### CODE REVIEW

**Comment by limpid-kzonix** [comment]
@lightbend-cla-validator, I signed the Lightbend CLA.

**Comment by mkurz** [comment]
This fix actually removes the tabs and just puts the Scala example underneath the Java one. That would work, however I would prefer if we keep the tabs and fix the rendering of the examples itself.

File: documentation/manual/releases/release27/migration27/Migration27.md [hunk 1]
```diff
@@ -199,14 +201,16 @@ If you are comfortable making some straightforward code changes and changing you
 
 To define the logger in each class, you can define:
 
-Java
-: ```java
+Java: 
+
+```java
```
Comment (mkurz) [inline documentation/manual/releases/release27/migration27/Migration27.md]: There is no need to fix this code snippets here. They look good, see https://www.playframework.com/documentation/2.7.0-RC9/Migration27#Static-Logger-singletons-deprecated.

Same is true for all other code snippets below.

Comment (limpid-kzonix) [reply documentation/manual/releases/release27/migration27/Migration27.md]: @mkurz, you are right. There was a problem with empty lines. I made changes. I tested it locally - looks fine.

**Comment by mkurz** [comment]
I am pretty sure that new line problem is a bug in pegdown [we use for parsing markdown](https://github.com/playframework/play-doc/blob/6b5c78829fb7d3b00b203e0eb469a3496350f539/build.sbt#L11). However pegdown [is end of life](https://github.com/sirthias/pegdown#-deprecation-note-) and should be replaced in play-doc one day.
(The bug is definitely not caused [by the code that creates the tabs](https://github.com/playframework/playframework/blob/2.7.0-RC9/documentation/style/main.js)).

The only (dirty) workaround - besides removing the new lines or extracing the code into own files - is to put non-breaking spaces into the empty lines, like unicode `U+00A0` (no-break space) or `U+FEFF` (zero width no-break space). Tried that and it worked.

**Pull request closed without merging**